src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -134,6 +134,8 @@
       <_Dependency Remove="System.ValueTuple"/>
       <_Dependency Remove="System.Security.AccessControl"/>
       <_Dependency Remove="System.Security.Principal.Windows"/>
+      <_Dependency Remove="System.Text.Encodings.Web" />
+      <_Dependency Remove="System.Text.Json" />
```

Comment:
I don't understand how these are part of this change

Reply:
I'm not entirely sure either, but the story I'm telling myself is that System.Text.Json is a transitive dependencies from the SourceLink.Tools package, and Text.Encodings.Web is transitive of that, and I'm the lucky winner of being the first to reference System.Text.Json in Roslyn.

Basically I added the package reference, built on the command line, and an error told me to modify this file :)